apps/address-service/index.js

```diff
@@ -63,7 +63,7 @@ module.exports = {
                 getAddressProviderLimitHealthCheck(),
             ],
         }),
-        conf.NODE_ENV === 'test' ? undefined : new OIDCKeystoneApp(),
+        conf.DISABLE_OIDC_AUTH ? undefined : new OIDCKeystoneApp(),
```

Comment:
do we need add new envs in `.env.example` ?

Reply:
I don't think so. This flag should be used in test environment only. But .env.example used to show how to startup project